Localize/lang/strings.json

```
@@ -7,6 +7,7 @@
   "+JCU+Y": "Select this if you are configuring an on-prem connection",
```

What does 'this" reference? Use "on-premises", not "on-prem". See ["on-premises" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/o/off-premises-on-premises).

```suggestion
  "+JCU+Y": "Select this if you are configuring an on-premises connection.",
```